### ISSUE TITLE
[FEAT]: add gurkify cmd to add gurkan to username

### ISSUE THREAD

**Issue by Vyvy-vi** [gurkult/gurkbot#35]
#### Description:

This command adds converts member.display_name into a gurkan-ified name. 
The current approach to this(as discussed on discord) would be:

---

<embed.color=green)>
:cucumber: **GURKAN NAMES**
**Top pick**
Vestergurkan

**Other options**
• Vesurkan
• Vestgurkan
• Gurkangurkan
• Vesturkan

---

**Comment by gustavwilliam** [comment]
I mentioned an approach to finding the name propositions earlier, on Discord. Do you have any plan on how to implement this? We don’t just want the “top pick” to be a randomly chosen one, but rather a name that should be one of the best.

**Comment by Vyvy-vi** [comment]
for the best selection, I think we might need to train an ml model..

**Comment by gustavwilliam** [comment]
I don’t think that’s necessary. There are certain patterns in the names and where certain characters are that indicate how the name should best be modified.

Either way, we could wait with the “top pick” for now, and just implement the basic name list, so we have something to work with.

**Comment by gustavwilliam** [comment]
Do you have any thoughts on how this could be done? What would be some ways to "gurkify" a name?

**Comment by Vyvy-vi** [comment]
we could do something like this(there might be better ways to do this tho):
```
1- split names into words(remove spaces)
2- try appending urkan or gurk to the received words
3- run this through test-samples(the top-pick checking logic)
4- if none of those seems to work, put gurkan in middle of the name.
5- Try different iterations with initials ended with 'urkan'
6- Try different iterations with gurk and ending letters of the name.
```

**Comment by onerandomusername** [comment]
Closed by #131
